src/mono/wasm/runtime/jiterpreter-opcodes.ts

```diff
@@ -0,0 +1,1833 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Keep this file in sync with mintops.def. The order and values need to match exactly.
+
+export const enum MintOpcode {
```

Comment:
`mintops.def` changes very often. Can't we do anything to simply include it ? Or have this automatically generated ?

Reply:
I think it should be possible for the runtime build to create a .ts file using some C preprocessor steps (or, worst case, using an MSBuild task). and then the wasm build can just consume the .ts file from elsewhere. 

> Note that if the mintops change is actually changing the implementation of an opcode, that'll break the jiterpreter unless it's also updated. Do you have thoughts on how that should be handled?

You mean if the semantics of an opcode change?  I think we should probably just handle it ad-hoc and disable the opcode.